src/packages/@ncigdc/components/UserDropdown.js

```diff
@@ -36,16 +37,22 @@ const DropdownItemStyled = styled(DropdownItem, {
 });
 
 const logout = () => {
+  if (IS_DEV) {
+    localStorage.setItem('ALLOW_FAKE_USER', '');
```

Comment:
now that it's `process.env.ALLOW_FAKE_USER` this wouldn't work anymore, it's fine no need to logout a fake user in dev; would just remove this block